File: DSA 1/Conditional Problem6.java
import java.util.*;

// https://course.acciojob.com/idle?question=f2a96172-f162-49c7-8ac9-f6f79a1c0fb5

public class Main {
    public static void main(String[] args) throws Throwable {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int m = sc.nextInt();
        sc.close();

		if(n%2!=0 && m%2!=0){
			System.out.println("we are odd");
		}else{
			System.out.println("we are simple");
		}
    }
}

File: Javascript/Conditional Problem 6.js
// https://course.acciojob.com/idle?question=3f36c2e7-2f78-4446-8a9c-c2de5ef059b2

const readline = require('readline');

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

rl.on('line', (input) => {
  const [a, b] = input.split(' ').map(Number);
  // write code here
	if(a%2 !=0 && b%2 != 0)
		console.log('we are odd')
	else
		console.log('we are simple')

  rl.close();
});
